charts/datadog/templates/_helm_check_config.yaml

```diff
@@ -1,6 +1,6 @@
 {{- define "helmCheck-config" -}}
 helm.yaml: |-
-{{- if .Values.datadog.clusterChecks.enabled }}
+{{- if and .Values.datadog.clusterChecks.enabled .Values.clusterChecksRunner.enabled}}
```

Comment:
you need a space at the end, else with helm2 it can be an issue (true story :) )
```suggestion
{{- if and .Values.datadog.clusterChecks.enabled .Values.clusterChecksRunner.enabled }}
```

Reply:
Fixed :+1: